src/main/kotlin/com/angorasix/projects/presentation/application/ProjectsPresentationService.kt

```diff
@@ -24,11 +25,21 @@ class ProjectsPresentationService(private val repository: ProjectPresentationRep
     suspend fun updateProjectPresentation(
         id: String,
         updateData: ProjectPresentation,
+        simpleContributor: SimpleContributor
```

Comment:
Not critical, but since we'll be making changes to this repo anyway (I have made a few more comments), let's change (across the board) this variable name that we pass around  to `requestingContributor` so that it's clearer, especially on lower-level layers/classes